thunder/executors/nvfuserex_impl.py

```diff
@@ -3188,6 +3188,35 @@ def argsort_transform(
 # Register argsort with NVFuser
 register_supported(prims.argsort, argsort_transform, _argsort_check_)
 
+
+def _grouped_mm_check_(
+    a: TensorProxy,
+    b: TensorProxy,
+    offs: TensorProxy,
+    bias: TensorProxy = None,
+    out_dtype=None,
+) -> bool:
+    return True
+
+
+def _grouped_mm_transform(
+    a: TensorProxy,
+    b: TensorProxy,
+    offs: TensorProxy,
+    bias: TensorProxy = None,
+    out_dtype=None,
+    *,
+    fd: FusionDefinition,
+    lc_to_nv_map: dict,
+) -> list[TensorLike]:
+    nva = getnv(a, fd, lc_to_nv_map)
+    nvb = getnv(b, fd, lc_to_nv_map)
+    nvoffs = getnv(offs, fd, lc_to_nv_map) if offs is not None else None
+    return fd.ops.grouped_mm(nva, nvb, nvoffs)
```

Comment:
it seems that `bias` is not used.
Could you make use of it if it's supported or update the checker to return when `bias is not None`?

Reply:
I'm afraid I don't think this comment is resolved. What's the plan about `bias` and `out_dtype`?

Reply:
I checked bias and out_dtype in torch/__init__.py. Is that sufficient?